implementations/rust/ockam/ockam_api/src/cli_state/cli_state.rs

```diff
@@ -315,6 +351,18 @@ pub fn random_name() -> String {
     petname::petname(2, "-").unwrap_or(hex::encode(random::<[u8; 4]>()))
 }
 
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum CliStateMode<'a> {
+    Persistent(Cow<'a, PathBuf>),
```

Comment:
Can we just clone it?

Reply:
Done. I was trying to save some clones, but it wasn't really helpful.